.github/workflows/lint.yml

```diff
@@ -2,32 +2,41 @@ name: lint
 
 on: push
 
+env:
+  POETRY_VERSION: "1.3.1"
+  PYTHON_VERSION: "3.9"
```

Comment:
Potentially not for now, but it might be reasonable to move to `3.10` - we have done this by default in the `cookiecutters`.

The python version in the tomlfile is very permissive (which makes sense if we want to make sure people utilise it). To be absolutely correct we should perhaps test against multiple python versions, but that would require quite a re-jig.

Reply:
Agreed probably should be probably upgrade to 3.10 on another MR. And yes I think testing on at least 3.9 and 3.10 would be a good change.